demo-app/src/main/java/org/openmined/syft/demo/federated/ui/MnistViewModel.kt

```diff
@@ -39,7 +41,11 @@ class MnistViewModel(
         get() = _processData
     private val _processData = MutableLiveData<ProcessData>()
 
+    private val result = mutableListOf<Float>()
+
     fun startCycle() {
+        syftWorker = Syft.getInstance(configuration, authToken)
```

Comment:
Could `syftWorker` and `mnistJob` be passed as dependencies?

Reply:
I want them to recreated everytime

Reply:
Also, this seems to be a temporary mechanism. We should actually not create syft workers again and again but to test without getting rejected by Pygrid, I didn't have any other choice